diff --git a/node_modules/react-native-reanimated/lib/reanimated1/animations/timing.js b/node_modules/react-native-reanimated/lib/reanimated1/animations/timing.js
index 7b6f442..4835f2a 100644
--- a/node_modules/react-native-reanimated/lib/reanimated1/animations/timing.js
+++ b/node_modules/react-native-reanimated/lib/reanimated1/animations/timing.js
@@ -50,7 +50,7 @@ const internalTiming = proc(function (
   ]);
 });
 
-export default function (clock, state, config) {
+export default function Timing (clock, state, config) {
   if (config.duration === 0) {
     // when duration is zero we end the timing immediately
     return block([set(state.position, config.toValue), set(state.finished, 1)]);
diff --git a/node_modules/react-native-reanimated/lib/reanimated2/js-reanimated/JSReanimated.js b/node_modules/react-native-reanimated/lib/reanimated2/js-reanimated/JSReanimated.js
index c3962dc..84fd10d 100644
--- a/node_modules/react-native-reanimated/lib/reanimated2/js-reanimated/JSReanimated.js
+++ b/node_modules/react-native-reanimated/lib/reanimated2/js-reanimated/JSReanimated.js
@@ -10,10 +10,12 @@ export default class JSReanimated {
         this._renderRequested = false;
         this._mapperRegistry = new MapperRegistry(this);
         this._frames = [];
-        this.timeProvider = null;
-        this.timeProvider = process.env.JEST_WORKER_ID
-            ? { now: () => Date.now() }
-            : window.performance;
+        this.timeProvider = {};
+        if(process.env.JEST_WORKER_ID) {
+          this.timeProvider.now = () => Date.now();
+        } else {
+          this.timeProvider.now = () => window.performance.now();
+        }
     }
     pushFrame(frame) {
         this._frames.push(frame);
